lib/i2c.ex

```diff
@@ -227,4 +227,25 @@ defmodule Circuits.I2C do
       result -> result
     end
   end
+
+  defmodule :circuits_i2c do
+    @doc """
+    Provide an Erlang friendly interface to Circuits
+    Example Erlang code:  circuits_i2c:open("i2c-1")
```

Comment:
Does this work? I think our open requires a binary instead of a list. Fwiw, I think our open should take both Elixir and Erlang strings.

Reply:
Yes this works.  open() calls to_charlist() before calling the NIF function, so it doesn't matter if a binary or list is passed to open().

Reply:
Ahh, yeah, I see that now. Thanks.